modules/tinymce/src/core/main/ts/api/dom/ScriptLoader.ts

```diff
@@ -111,6 +116,11 @@ class ScriptLoader {
         dom.setAttrib(elm, 'referrerpolicy', this.settings.referrerPolicy);
       }
 
+      const crossOrigin = this.settings.crossOrigin;
+      if (Type.isString(crossOrigin) && crossOrigin.length > 0) {
```

Comment:
The `length > 0` check here is the only place you acknowledge that the default value is not one of the values allowed by the option type 🤔